orte/mca/schizo/ompi/schizo_ompi.c

```diff
@@ -75,96 +75,109 @@ orte_schizo_base_module_t orte_schizo_ompi_module = {
 
 static opal_cmd_line_init_t cmd_line_init[] = {
     /* Various "obvious" options */
-    { NULL, 'h', NULL, "help", 0,
-      &orte_cmd_options.help, OPAL_CMD_LINE_TYPE_BOOL,
-      "This help message" },
+    { NULL, 'h', NULL, "help", 1,
+      &orte_cmd_options.help, OPAL_CMD_LINE_TYPE_STRING,
+      "This help message.  Argument options are:\n\t\t\t general - General arguments (Defaults to this option)\n\t\t\t debug - Arguments usefull for debugging\n\t\t\t output - Arguments to modify output options\n\t\t\t input - Arguments to modify input options\n\t\t\t mapping - Mapping arguments\n\t\t\t ranking - Ranking arguments\n\t\t\t binding - Binding arguments\n\t\t\t devel - Arguments usefull to OMPI Developers\n\t\t\t compatibility - Arguments supported for backwards compatibility\n\t\t\t launch - Arguments to modify launch options\n\t\t\t dvm - Distributed Virtual Machine arguments", OPAL_CMD_LINE_OTYPE_GENERAL },
```

Comment:
This message still needs to be cleaned up, but I wanted to get the everything up so people could look at it and comment on it.